apps/api/tests/endpoints/test_document_likes.py

```
@@ -0,0 +1,155 @@
+
+class DocumentLikeEndpointsTestCase(support.SharedAppTestCase):
+    """Thumbs-up likes held as ``(:User)-[:LIKED]->(:Document)`` edges."""
+
+    def setUp(self) -> None:
+        from imbi.api.auth import permissions
+        from imbi.common import graph
+
+        self.user = models.User(
+            email='reader@example.com',
+            display_name='Reader',
+            password_hash='$argon2id$hashed',
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to argument: "password_hash"

(S106)